OptionSelector/CategoryTableViewController.m

```diff
@@ -0,0 +1,118 @@
+//
+//  CategoryTableViewController.m
+//  OptionSelector
+//
+//  Created by Henna on 8/10/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "CategoryTableViewController.h"
+#import "CQCategory.h"
+#import "OptionsTableViewController.h"
+#import "CategoryModel.h"
+
+
+@interface CategoryTableViewController ()
+@property (nonatomic) CategoryModel * data;
+
+@end
+
+@implementation CategoryTableViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    self.title = @"Henna's Categories";
+    self.data = [CategoryModel sharedInstance];
+    [self.data initializeModel];
+    
+}
+
+- (void)viewDidAppear:(BOOL)animated {
+[self.tableView reloadData];
+}
+
+
+- (void)didReceiveMemoryWarning {
+    [super didReceiveMemoryWarning];
+    // Dispose of any resources that can be recreated.
+}
+
+#pragma mark - Table view data source
+
+- (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView {
```

Comment:
Style: The [default return value of this method is 1](https://developer.apple.com/library/ios/documentation/UIKit/Reference/UITableViewDataSource_Protocol/#//apple_ref/occ/intfm/UITableViewDataSource/numberOfSectionsInTableView:), so you can also remove this method definition if you want.